src/lib/CanvasWorld.js

```diff
@@ -43,6 +43,24 @@ export default class CanvasWorld {
     ];
   }
 
+  /** */
+  canvasToWorldCoordinates(canvasId) {
```

Comment:
This method is very similar to the one above it. I think this is ok, as we may want to support fragment placement, rotation, and things like that which may mean we will likely expand the scope of either of these.